KinoG/KinoG/App/Flows/SelectedFilmModule/View/Cell/SelectedFilmTableViewCell.swift

```diff
@@ -10,40 +10,41 @@ final class SelectedFilmTableViewCell: UITableViewCell {
         image.backgroundColor = .blue
         return image
     }()
-    
+
     private var filmNameLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .center
         label.numberOfLines = 10
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
-    
+
     private var filmDiscriptionLabel: UILabel = {
         let label = UILabel()
         label.numberOfLines = 0
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
-    
+
     private var filmDateLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .center
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
+
     // MARK: - Private Properties
-    
+
     private let imageCatalogAdress = "https://image.tmdb.org/t/p/w500/"
```

Comment:
```suggestion
    private let imageCatalogAdress = "https://image.tmdb.org/t/p/w500/"
```
в сетевой в константу вынеси